doc/source/user_guide/ug_case_settings.rst

```diff
@@ -8,7 +8,52 @@ Case Settings
 There are two important files that define the case, **cice.settings** and 
 **ice_in**.  **cice.settings** is a list of env variables that define many
 values used to setup, build and run the case.  **ice_in** is the input namelist file
-for CICE.  Variables in both files are described below.
+for CICE.  Variables in both files are described below.  In addition, the first
+table lists available preprocessor directives to activate or deactivate various
+features when compiling.
+
+.. _tabcpps:
+
+Table of C Preprocessor (CPP) Directives
+---------------------------------------------------
+
+The CICE model supports a number of C Preprocessor (CPP) Directives.  These
+can be turned on during compilation to activate different pieces of source
+code.  The main purpose is to introduce build-time code modifications to
+include or exclude certain libraries or Fortran language features.  More information
+can be found in :ref:`cicecpps`.  The following CPPs are available.
+
+.. csv-table:: **CPP directives**
+   :header: "CPP name", "description"
+   :widths: 15, 60
+
+   "",""
+   "**General Directives**", ""
+   "CESM1_PIO", "Provide backwards compatible support for PIO interfaces/version released with CESM1 in about 2010"
+   "coupled", " "
```

Comment:
is this intentional to not describe "coupled" ?

Reply:
Yes, because I don't know what to say.  It does a bunch of things.  I have already started working on a new PR that removes the coupled CPP flag following some discussion in #288.